neo/Prompt/Commands/Config.py

```diff
@@ -0,0 +1,205 @@
+        c1 = get_arg(arguments)
+        if c1 is not None:
+            c1 = c1.lower()
+            if c1 == 'on' or c1 == '1':
+                settings.set_emit_notify_events_on_sc_execution_error(True)
+                print("Smart contract emit Notify events on execution failure is now enabled")
+                return c1
+            elif c1 == 'off' or c1 == '0':
+                settings.set_emit_notify_events_on_sc_execution_error(False)
+                print("Smart contract emit Notify events on execution failure is now disabled")
+                return c1
+            else:
+                print("Cannot configure log. Please specify on|off")
+                return
+        else:
+            print("Cannot configure log. Please specify on|off")
+            return
+
+    def command_desc(self):
+        p1 = ParameterDesc('attribute', 'either "on"|"off" or 1|0')
+        return CommandDesc('sc-debug-notify', 'toggle Notify events on execution failure on or off', [p1])
+
+
+class CommandConfigVMLog(CommandBase):
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        c1 = get_arg(arguments)
+        if c1 is not None:
+            c1 = c1.lower()
+            if c1 == 'on' or c1 == '1':
+                settings.set_log_vm_instruction(True)
+                print("VM instruction execution logging is now enabled")
+                return c1
+            elif c1 == 'off' or c1 == '0':
+                settings.set_log_vm_instruction(False)
+                print("VM instruction execution logging is now disabled")
+                return c1
+            else:
+                print("Cannot configure VM instruction logging. Please specify on|off")
+                return
+        else:
+            print("Cannot configure VM instruction logging. Please specify on|off")
+            return
+
+    def command_desc(self):
+        p1 = ParameterDesc('attribute', 'either "on"|"off" or 1|0')
+        return CommandDesc('vm-log', 'toggle VM instruction execution logging on or off', [p1])
+
+
+class CommandConfigNodeRequests(CommandBase):
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        if len(arguments) in [1, 2]:
+            if len(arguments) == 2:
+                try:
+                    return NodeLeader.Instance().setBlockReqSizeAndMax(int(arguments[0]), int(arguments[1]))
+                except ValueError:
+                    print("Please specify a block request part and max size for each node, like 30 and 1000")
+                    return False
+            elif len(arguments) == 1:
+                return NodeLeader.Instance().setBlockReqSizeByName(arguments[0])
+        else:
+            print("Invalid number of arguments")
+            return False
+
+    def command_desc(self):
+        p1 = ParameterDesc('attribute', 'specify "slow"/"normal"/"fast" preset block request sizes, or specific block request size')
```

Comment:
I understand why you used `attribute` but I think for this case `block-size` could work better. After reading the help output it just felt weird. 
```suggestion
         p1 = ParameterDesc('block-size', 'a preset of "slow"/"normal"/"fast", or a specific block request size (max. 500) e.g. 250 ')
```